### ISSUE TITLE
Nice idea - didn't work

### ISSUE THREAD

**Issue by Dan-Essin** [a2o/snoopy#257]
## Checklist before starting to submit this support request

I confirm that:
- [X] I am requesting support! :)
- [X ] I have tested this with the [latest stable Snoopy version](https://github.com/a2o/snoopy/releases/latest) (or the latest master build).
- [X ] I have checked the [FAQ](https://github.com/a2o/snoopy/blob/master/doc/FAQ.md).
- [ X] I have read Snoopy's documentation [here](https://github.com/a2o/snoopy/blob/master/README.md) and [here](https://github.com/a2o/snoopy/tree/master/doc).
- [ X] I have searched [Snoopy issues](https://github.com/a2o/snoopy/issues?q=is%3Aissue) for an existing issue that matches my problem, and found none.

## Issue description

On Fedora 37, Installed with dnf version 2.4.14
used default ini file.
rebooted
/etc/dev empty
added export LD_PRELOAD
rebooted 
/etc/dev empty
edited ini file, remove ; from devlog
rebooted
/etc/dev empty

## Expected result

I thought I would get some output

## Actual result

Got no output

It won't be much use if it doesn't generate output.
I've obviously missed something, although the tutorials suggest that it should just install and work.

btw - the github MD file forgot to mention snoopy-enable and snoopy-disable. What other critical piece of information was also missing?

**Comment by bostjan** [comment]
Hey @Dan-Essin, for anything packaged by distribution maintainers, talk to them first. In this case, this is (apparently) someone that goes by the username `ngompa` (mentioned [here](https://packages.fedoraproject.org/pkgs/snoopy/snoopy/)). There are various things the packager may have done/tweaked when building a package (like disabling the config file altogether), so please seek Fedora-specific support there.

Since v2.5.0, this project has started providing upstream-packaged binaries for a select set of distributions. Therefore, and if you can, you may want to try that option, i.e. the AlmaLinux (a RHEL clone) package. You can find more info [here](https://github.com/a2o/snoopy/blob/master/doc/install-from-repo.md#almalinux-os).

As a last resort, you can try building Snoopy from the source code (via our provided `install-snoopy.sh` file, [here](https://github.com/a2o/snoopy/blob/master/doc/install-from-source.md)).

PS: In the latest Snoopy versions (since 2.5.0), `snoopy-enable` and `snoopy-disable` commands were superseded by a unified tool called `snoopycfg`, and all the references to the old tool names were removed.

**Comment by flakshack** [comment]
Thanks for this post, I was missing that I needed to run snoopy-enable.
